### PR TITLE
autopkgtest subunit output

### PR DESCRIPTION
**Pull request by fgimenez**

### CODE REVIEW

File: containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml [hunk 1]
```diff
@@ -112,7 +117,7 @@ if [[ "$ghprbTargetBranch" = "15.04" ]]; then
   exit 0
 fi
 
-. $HOME/.openstack/novarc-lcy01
```
Comment (elopio) [inline containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml]: why did you reverted this one?

File: containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml [hunk 2]
```diff
@@ -10,6 +10,12 @@
   </logRotator>
   <keepDependencies>false</keepDependencies>
   <properties>
+    <hudson.plugins.buildblocker.BuildBlockerProperty plugin="build-blocker-plugin@1.7.3">
+      <useBuildBlocker>false</useBuildBlocker>
+      <blockLevel>GLOBAL</blockLevel>
+      <scanQueueFor>DISABLED</scanQueueFor>
+      <blockingJobs></blockingJobs>
+    </hudson.plugins.buildblocker.BuildBlockerProperty>
```
Comment (elopio) [inline containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml]: I don't understand what this is for.

File: containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml [hunk 3]
```diff
@@ -121,18 +126,19 @@ openstack keypair create --public-key /home/jenkins-slave/.ssh/id_rsa.pub $BUILD
...
-adt-run .// --setup-commands setup-testbed -o $WORKSPACE/adt-run-output --- ssh -d -l ubuntu -o='-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no' -s nova -- -d --flavor autopkgtest --image 'ubuntu/ubuntu-xenial-16.04-amd64-server-*' --keyname $BUILD_TAG --net-id=net_ues-snappy-integration --security-groups ssh -e 'http_proxy=http://squid.internal:3128' -e 'https_proxy=http://squid.internal:3128' -e 'no_proxy=127.0.0.1,127.0.1.1,localhost,localdomain,novalocal,internal,archive.ubuntu.com,security.ubuntu.com,ddebs.ubuntu.com,ppa.launchpad.net'
+adt-run .// --setup-commands setup-testbed -o $WORKSPACE/adt-run-output --- ssh -d -l ubuntu -o=&apos;-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no&apos; -s nova -- -d --flavor autopkgtest --image &apos;ubuntu/ubuntu-xenial-16.04-amd64-server-*&apos; --keyname $BUILD_TAG --net-id=net_ues-snappy-integration --security-groups ssh -e &apos;http_proxy=http://squid.internal:3128&apos; -e &apos;https_proxy=http://squid.internal:3128&apos; -e &apos;no_proxy=127.0.0.1,127.0.1.1,localhost,localdomain,novalocal,internal,archive.ubuntu.com,security.ubuntu.com,ddebs.ubuntu.com,ppa.launchpad.net&apos;
```
Comment (elopio) [inline containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml]: are these &apos; needed?

File: containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml [hunk 4]
```diff
@@ -121,18 +126,19 @@ openstack keypair create --public-key /home/jenkins-slave/.ssh/id_rsa.pub $BUILD
...
+
+subunit2junitxml $WORKSPACE/adt-run-output/artifacts/results.subunit -o $WORKSPACE/adt-run-output/artifacts/results.xml
 ]]></command>
     </hudson.tasks.Shell>
   </builders>
   <publishers>
-    <hudson.tasks.ArtifactArchiver>
-      <artifacts>adt-run-output/binaries/*.deb</artifacts>
```
Comment (elopio) [inline containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml]: I've found the debs to be useful a couple of times.

File: containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml (hunk 2 above)
Comment (fgimenez) [inline containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml]: Nor do I, I copied the config from the running server, will remove it

File: containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml (hunk 3 above)
Comment (fgimenez) [inline containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml]: nope, removing

File: containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml (hunk 1 above)
Comment (fgimenez) [inline containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml]: same as above, fixing

File: containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml (hunk 4 above)
Comment (fgimenez) [inline containers/jenkins-master/config/jobs/github-snappy-autopkgtest-cloud/config.xml]: Not sure if we are going to have problems with the extended storage needed, anyway there are more people asking for it, i'll reenable them

**Pull request closed without merging**